src/system.ts

```diff
@@ -77,6 +80,18 @@ process.on('uncaughtException', (error) => {
 
 const arweave = Arweave.init({});
 const arIO = ArIO.init({ contractTxId: config.CONTRACT_ID });
+const awsClient =
+  config.AWS_ACCESS_KEY_ID !== undefined &&
+  config.AWS_SECRET_ACCESS_KEY !== undefined &&
+  config.AWS_REGION !== undefined
+    ? await awsLite({
+        accessKeyId: config.AWS_ACCESS_KEY_ID,
```

Comment:
We won't to be able to use instance roles, so let's allow for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` being `undefined`.

Reply:
@karlprieb there's some examples of setting up AWS configs for local use in the Turbo source. Not sure if totally relevant here:

https://github.com/ardriveapp/upload-service/blob/dev/src/arch/s3ObjectStore.ts#L60-L72
and
https://github.com/ardriveapp/upload-service/blob/dev/src/arch/s3ObjectStore.ts#L164-L187

Reply:
`aws-lite` throws if `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are not defined.
We can have another ticket to use instance profiles as they support it

Reply:
I looked into this a bit too after adding the comment. Seems like there's no cred provider integrated that'll use instance roles by default. I think we need to retrieve keys and then switch to the instance role, but that's more work than we want to do in this ticket.